routes/authRoutes.ts

```diff
@@ -11,6 +13,7 @@ router
     .route("/register")
     .post(
         protect({ loggedIn: false }),
+	    validateMiddleware(registerSchema),
```

Comment:
Make sure all of the functions are in line. validateMiddlware is indented further than the protect functions